torch/_tensor.py

```diff
@@ -383,14 +415,25 @@ def _reduce_ex_internal(self, proto):
             type(self) is not torch.Tensor
             and type(self).__torch_dispatch__ is not torch.Tensor.__torch_dispatch__
             and (
-                isinstance(
-                    self,
-                    (
-                        torch._subclasses.fake_tensor.FakeTensor,
-                        torch._subclasses.functional_tensor.FunctionalTensor,
-                    ),
+                (
+                    isinstance(
+                        self, torch._subclasses.functional_tensor.FunctionalTensor
```

Comment:
Should we just split functional + fake Tensor to another condition just above? I have to admit I can't read this condition anymore :D